code/modules/admin/verbs/individual_logging.dm

```diff
@@ -23,8 +23,9 @@
 	else
 		dat += "<center>[type] of [key_name(M)]</center><br>"
 		var/list/reversed = M.logging[type]
-		reversed = reverseRange(reversed.Copy())
-		for(var/entry in reversed)
-			dat += "<font size=2px>[entry]: [reversed[entry]]</font><hr>"
+		if(reversed)
```

Comment:
is it valid for it to be null at this point or is the bug that there are nulls in the logging[type] list